src/components/item/publish/CCLicenseSelection.js

```diff
@@ -115,24 +123,42 @@ const CCLicenseSelection = ({ item }) => {
           value={CC_LICENSE_ADAPTION_OPTIONS.ALLOW}
           control={<Radio color="primary" />}
           label={t('Yes')}
-          disabled={disabled}
         />
         <FormControlLabel
           value={CC_LICENSE_ADAPTION_OPTIONS.ALIKE}
           control={<Radio color="primary" />}
           label={t('Only if others share alike')}
-          disabled={disabled}
         />
       </RadioGroup>
       <CCLicenseDialog
         open={open}
         setOpen={setOpen}
-        disabled={disabled || !optionValue}
+        disabled={!optionValue}
         className={classes.button}
+        buttonName={t('Submit')}
+        dialogContent={t(
+          'Please verify that your item fits the CC License, and do not change to a more restricted option.',
+        )}
         handleSubmit={handleSubmit}
       />
+      {settings?.ccLicenseAdaption !== '' && (
+        <CCLicenseDialog
```

Comment:
I don't really think this works well. 
If `settings?.ccLicenseAdaption !== ''` is `true`, then you display both dialog when `open` becomes `true`.
Plus `settings?.ccLicenseAdaption !== ''` means "is my cc license is (already) empty" -> so you don't actually need to remove it? 

I think you need at least to variables to control `open` of each dialog. And have at least a `if/else` close to display only one of the dialog.